lib/koala/http_service.rb

```diff
@@ -116,6 +116,9 @@ def self.make_request(path, args, verb, options = {})
       # Log URL information
       Koala::Utils.debug "#{verb.upcase}: #{path} params: #{params.inspect}"
       Koala::HTTPService::Response.new(response.status.to_i, response.body, response.headers)
+
+    rescue Faraday::Error::ConnectionFailed => error
+      raise Koala::KoalaError::ConnectionFailed.new(error.to_s, error.response)
```

Comment:
I just realized that in addition to reraising it, we should copy the original backtrace as well. I'll add that as I merge it in.